### PR TITLE
Included all French translations

### PR DESCRIPTION
**Pull request by DRH-TBS**

### CODE REVIEW

File: openapi.yaml [hunk 1]
```diff
@@ -50,17 +97,51 @@ info:
...
+ 
+      Points terminaux en lecture seule (« *_montrer », « *_liste », « *_recherche », etc.) 
+      Accepter les demandes d’obtention de données (GET) avec des paramètres de requête. 
+ 
+Les réponses sont renvoyées dans une enveloppe JSON en cas de succès : 
+« `json 
+{ 
+  “aide” : “url pour compléter l’aide sur les points terminaux”, 
+  “succès” : vrai, 
+  “résultat” : {…}
```
Comment (wardi) [inline openapi.yaml]: the keys in these examples can't be translated, and the quotes need to be ascii `"` quotes

File: openapi.yaml [hunk 2]
```diff
@@ -50,17 +97,51 @@ info:
...
+  “aide” : “url pour compléter l’aide sur les points terminaux”, 
+  “succès” : vrai, 
+  “résultat” : {…} 
+} 
+ » 
+Ou sur les erreurs : 
+« `json 
+{ 
+  ‘aide’ : ‘url pour compléter l’aide sur les points terminaux’,
+  ‘succès’ : faux,
```
Comment (wardi) [inline openapi.yaml]: true/false/null also can't be translated

File: openapi.yaml [hunk 3]
```diff
@@ -85,35 +166,71 @@ tags:
...
+      summary: 
+        en: |
+          Get a list of all package ids (datasets)
+        fr: |
+          Obtenir une liste de tous les identifiants des paquetages (jeux de données). 
+      description:
+        en: |
+          Note: our site uses the same value for the "name" and "id" fields of datasets
+        fr: |
+          Remarque : notre site utilise la même valeur pour les champs « nom » et « identifiant » des jeux de données
```
Comment (wardi) [inline openapi.yaml]: field names can't be translated, or maybe there's a way to provide both the field names and their translations in this text like `les champs « name » (nom) et « id » (identifiant)`? This is a question for the translation team.

File: openapi.yaml [hunk 4]
```diff
@@ -184,18 +325,37 @@ paths:
...
+          Obtenir les métadonnées afférentes à un paquetage particulier (jeu de données). 
+      description: 
+        en: |
+          Returns metadata about the package (dataset) corresponding to its UUID.
+          Refer to the [open data schema (type=dataset)](https://open.canada.ca/data/api/action/scheming_dataset_schema_show?type=dataset)
+          or the [open information schema (type=info)](https://open.canada.ca/data/api/action/scheming_dataset_schema_show?type=info)
+          for information about the metadata fields returned.
+        fr: |
+          Renvoie les métadonnées du paquetage (jeu de données) correspondant à son IDUU (identifiant unique universel).
+          Se référer au [schéma de données ouvert (type=jeu de données)] (https://open.canada.ca/data/api/action/scheming_dataset_schema_show?type=dataset)
```
Comment (wardi) [inline openapi.yaml]: same issue here, can we still say `type=dataset` and provide the translation at the same time?

File: openapi.yaml (hunk 1 above)
Comment (DRH-TBS) [reply openapi.yaml]: Quotes are now ascii quotes, keys have been updated

File: openapi.yaml (hunk 2 above)
Comment (DRH-TBS) [reply openapi.yaml]: French values updated

File: openapi.yaml (hunk 3 above)
Comment (DRH-TBS) [reply openapi.yaml]: Applied recommendation

File: openapi.yaml (hunk 4 above)
Comment (DRH-TBS) [reply openapi.yaml]: Applied recommendation

**Comment by JVickery-TBS** [comment]
@wardi So these are just translations for our descriptions etc. The other parts of the UI have yet to be translated.

E.g.
* Parameters
* Responses
* Code
* Try it out

Do you think these will have to be done with a po file? Or is there some other Swagger UI i18n support?

**Review by wardi — APPROVED** [review]

**Pull request merged**